man/file-hierarchy.xml

```diff
@@ -589,6 +589,18 @@
     directives of service units (see
     <citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry>
     for details).</para>
+
+    <para><filename>/tmp/</filename>, <filename>/var/tmp/</filename> and <filename>/dev/shm/</filename>
+    should be mounted <option>nosuid</option> and <option>nodev</option>, which means that set-user-id mode
+    and character or block special devices are not interpreted on those file systems. But they should
+    <emphasis>not</emphasis> not be mounted <option>noexec</option>, because various programs use those
```

Comment:
maybe say that *generally* they shouldn't be mounted like that. In some special cases they might be and it would be OK if the vendor made sure no code that wants to map stuff executable is ever installed.

key really is that in the general case it's not OK, in specific cases it might very well be though

Reply:
The new version is still a bit weaker than I'd prefer, but I suppose it's better than before, so LGTM.